app/src/main/java/io/homeassistant/companion/android/sensors/SensorsSettingsFragment.kt

```diff
@@ -184,10 +207,75 @@ class SensorsSettingsFragment : PreferenceFragmentCompat() {
                 prefCategory.addPreference(pref)
             }
         }
+
+        if (showOnlyEnabledSensors) showHideGroupsIfNeeded()
+    }
+
+    private fun getShowOnlyEnabledSensors(showOnlyEnabledSensors: Boolean) {
+        runBlocking {
+            integrationUseCase.setShowOnlyEnabledSensors(showOnlyEnabledSensors)
+        }
+    }
+    private fun getShowOnlyEnabledSensors(): Boolean {
+        var showOnlyEnabledSensors = false
+        runBlocking {
+            showOnlyEnabledSensors = integrationUseCase.getShowOnlyEnabledSensors()
+        }
+        return showOnlyEnabledSensors
+    }
```

Comment:
I don't think we should store this as a setting.  This is just really a filter and should default to off but not be stored.

Reply:
I thought of this more like a setting rater than a temp. filter. My use case is, that i configure my sensors only one time (after app installation) and then work (check sensors states/reconfigure enabled sensors) from time to time with the sensors. In this use case it is really convenient to just see the enabled sensors, every time i enter the manage sensor screen.

Nevertheless, I like the temp. filter idea. But then i would not use a `SwitchPreference` to filter things. 
I imagine that in the upper right corner there is a filter symbol and a search symbol to find certain sensors.
But to realize this, i need to rework this PR. I think this is the better approach.

Reply:
I was under the impression that preferences are always stored and remember what the user selected/typed.  When I check notification history and change the filter the next time I come back it remembers it.  Even after I force stop the app and reopen.

Reply:
That's interesting. I tried it, without storing a setting, and the setting was reset every time i entered the manage sensor screen. I need to have a look at this again.

Reply:
Ok i reworked the PR a bit. (See post below) But i still think, its better to save the filters. I think some users would complain about the reset filter. For me it feels not right, when the filter gets reset every time.

@dshokouhi You were right. The preference save it's value automatically. But, now that i use the action bar menu items, i need to use the storage.